src/modules/mc_pos_control/PositionControl.hpp

```diff
@@ -77,12 +77,20 @@ struct Constraints {
  *
  * 	A setpoint that is NAN is considered as not set.
  */
-class PositionControl
+class PositionControl : public ModuleParams
 {
 public:
 
-	PositionControl();
-	~PositionControl() {};
+	PositionControl(ModuleParams *parent);
+	~PositionControl() = default;
+
+	/**
+	 *	Overwrites certain parameters.
+	 *	Overwrites are required for unit-conversion.
+	 *	This method should only be called if parameters
+	 *	have been updated.
+	 */
+	void overwriteParams();
```

Comment:
Can you make `ModuleParams::updateParams` virtual, and override it here instead?

Reply:
sure, that could work as well